Door

Inputusernamee opened the pull request:
This is the code and stuff for door. We kinda need the scenes just because the code is reliant on having a canvas with a black image, but then again, we just can make new ones. Also remember we have to put in the coordinates and scene name and other stuff for it to load properly.

Inputusernamee committed:
doors

tried to make doors

Inputusernamee committed:
door with code

i wrote some code

Inputusernamee committed:
Player too John Square

Changed player in my code to john square

Inputusernamee committed:
Code Change

I changed the code so it is easier to select the door that you will be transported to.

Inputusernamee committed:
TriggerTest

The trigger test didnt work so idk

Inputusernamee committed:
Scene transition

The door fricken works lets go!!!

Inputusernamee committed:
Position change

The character position isnt always where I want it sometimes causing problems such as too much loading at once and crashing the game

Inputusernamee committed:
Finally Doors

The doors that we needed.
Simplified the code enough that they can actually work.

Inputusernamee committed:
E to enter

Changed script so it only changes the scene and your location after pressing e

Inputusernamee committed:
Added Fade Transition

I added a transition so when you go through the door it fades out and then fades back in on the new scene. Huzzah!

Inputusernamee committed:
Moved some files

Just put files together for ease of use

Inputusernamee commented:
I dont understand merge conflicts?

Inputusernamee committed:
Merge conflict resolution?

Inputusernamee commented:
Matt help me i dont know how to fix this

Inputusernamee committed:
Fixed Conflicts

errors with samplescene

Inputusernamee committed:
Update Scene1.unity

Inputusernamee committed:
Update Scene1.unity

Inputusernamee committed:
removed SampleScene

Inputusernamee committed:
Added back scene1

Inputusernamee committed:
Bah

i hope

Inputusernamee committed:
scene

please

Inputusernamee committed:
Update startscene.unity

Gregularity commented:
Looks good to me, I don't believe it will conflict with anything when we merge.

Gregularity reviewed (approved):
I Approve; we will merge this branch immediately after merging the dialogue branch and testing it.

Inputusernamee committed:
Fixed merge conflicts

Added files located on Main but not on Door to Door in order to satisfy some of the conflicts. Updated the tag manager to contain both tags from Main and Door.

Inputusernamee commented:
Merge Complete
